backend/src/main/java/com/google/rolecall/Constants.java

```diff
@@ -32,11 +32,11 @@
 
   public static class Roles {
     public static final String ADMIN = "ADMIN";
-    public static final String COREOGRAPHER = "COREOGRAPHER";
```

Comment:
Is there any impact on the DB from this change?

Reply:
No. It seems as if the original developers changed strategy at some point and ignored their system wide constants. CHOREOGRAPHER only shows up in one place, inside a comment!